docs/reference/generated/kube-proxy.md

```diff
@@ -25,7 +25,7 @@ kube-proxy
 
 ```
       --azure-container-registry-config string       Path to the file container Azure container registry configuration information.
-      --bind-address ip                              The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces) (default 0.0.0.0)
+      --bind-address ip                              The IP address for the proxy server to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 0.0.0.0)
```

Comment:
should be fixed here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L128